### PR TITLE
Mention Vue and HTML linting support

### PR DESCRIPTION
**Pull request by MariaSolOs**
This PR updates the JavaScript linting documentation to also mention the newly integrated support for linting Vue and HTML files.

### CODE REVIEW

**Comment by Court72** [comment]
@Mikejo5000 

Can you review the proposed changes? *IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by Mikejo5000 — APPROVED** [review]
Looks good. Thanks, Maria!

**Comment by Mikejo5000** [comment]
#sign-off

File: docs/javascript/linting-javascript.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To enable linting support in Visual Studio 2022 or later versions, enable the **
 
 :::image type="content" source="media/vs-2022/linting-tools-options.png" alt-text="Screenshot of linting Tools Options page.":::
 
-In the options page, you can also modify the set of files that you want to lint. By default, all file extensions that can be linted (*.js*, *.jsx*, *.ts*, *.tsx*) will be linted.
+In the options page, you can also modify the set of files that you want to lint. By default, all file extensions that can be linted (*.js*, *.jsx*, *.ts*, *.tsx*, *.vue*, *.html*) will be linted. Note that for linting Vue and HTML files, the HTML LSP-based editor must be enabled (the respective setting can be found in **Tools > Options > Environment > Preview Features**).
```
Comment (Jak-MS) [inline docs/javascript/linting-javascript.md:28]:
```suggestion
In the options page, you can also modify the set of files that you want to lint. By default, all file extensions that can be linted (*.js*, *.jsx*, *.ts*, *.tsx*, *.vue*, *.html*) will be linted. The HTML LSP-based editor must be enabled for linting Vue and HTML files. The respective setting can be found in **Tools > Options > Environment > Preview Features**.
```
MS Style tweaks

**Pull request merged**